src/Printer/FormatPerservingPrinter.php

```diff
@@ -23,16 +23,13 @@ public function __construct(BetterStandardPrinter $betterStandardPrinter)
      * @param Node[] $oldStmts
      * @param Node[] $oldTokens
      */
-    public function printToFile(SplFileInfo $fileInfo, array $newStmts, array $oldStmts, array $oldTokens): bool
+    public function printToFile(SplFileInfo $fileInfo, array $newStmts, array $oldStmts, array $oldTokens): string
     {
-        $oldContent = file_get_contents($fileInfo->getRealPath());
         $newContent = $this->printToString($newStmts, $oldStmts, $oldTokens);
 
-        if ($oldContent === $newContent) {
-            return false;
-        }
+        file_put_contents($fileInfo->getRealPath(), $newContent);
 
-        return (bool) file_put_contents($fileInfo->getRealPath(), $newContent);
+        return $newContent;
```

Comment:
@carusogabriel Just to make it consistent with the other method to make 1 way to work with.